core/Pos/Core/Types.hs

```diff
@@ -102,6 +103,15 @@ import           Pos.Data.Attributes        (Attributes)
 -- Address
 ----------------------------------------------------------------------------
 
+-- | Hash used to identify address.
+type AddressHash = AbstractHash Blake2b_224
+
+-- | Stakeholder identifier (stakeholders are identified by their public keys)
+type StakeholderId = AddressHash PublicKey
+
+-- | Alias for set of stakeholders.
+type Stakeholders = HashSet StakeholderId
```

Comment:
Ok, makes sense as well. It's quite common to use `HashSet`, but i guess it's better to replace type alias with real type everywhere.